uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -175,6 +192,29 @@ define([
       };
 
       /**
+       * Format a date-time into user format and returns
+       *
+       * @return {String}
+       */
+      this.formatDateTime = function (dateTime) {
+        return moment(dateTime, sharedSettings.serverDateTimeFormat).format(this.uiOptions.userDateFormat + ' HH:mm');
+      };
+
+      /**
+       * Returns the comment author name
+       * @param {String} contact_id
+       *
+       * @return {String}
+       */
+      this.getCommentorName = function (contact_id) {
+        if(contact_id == this.directiveOptions.contactId) {
```

Comment:
No here the logged in users contact ID should be used. The leave request might be someone else's but the comment should be added for the currently logged in user.